packages/api/helper/scheduler.js

```diff
@@ -0,0 +1,11 @@
+const CronJob = require('cron').CronJob
+const { checkApplicationStatuses } = require('../controllers/nomination')
+// To override in testing set: EMAIL_CRON='*/45 * * * * *' runs every minute on the 45th second
+const defaultCron = process.env.EMAIL_CRON ?? '00 00 11 * * 0-6'
```

Comment:
The default is to run at midnight on the 11th day of every month? Maybe something like `0 12 * * * *` would be appropriate? 

I might also name this env var something a little clearer, like `EMAIL_REMINDER_CRON`.

This might also be a little wonky on Heroku, especially if you scale up to more than one API node. Alternatively, we could skip the cron entirely, just call `checkApplicationStatuses` directly from this script, and call it using the Heroku Scheduler.

Reply:
I believe the set up is: [Seconds] [Minutes] [Hours] [Day of the Month] [Months] [Day of the Week] (From: https://www.npmjs.com/package/cron)

So that schedule should be every day S-Sat [0-6] at 11am (CST). The time was set by Bill to be at 9am PST so folks on the west coast can have ample time to see the reminder email and turn in documents.

Reply:
Oh cool. My mistake, the seconds field isn't usually available in crontabs, so that threw me. Sounds good!